test/e2e-common.sh

```diff
@@ -345,8 +345,10 @@ function knative_teardown() {
 function test_setup() {
   echo ">> Setting up logging..."
 
-  # Install kail.
-  bash <( curl -sfL https://raw.githubusercontent.com/boz/kail/master/godownloader.sh) -b "$GOPATH/bin"
+  # Install kail if needed.
+  if [[ ! -x "$GOPATH/bin/kail" ]]; then
```

Comment:
do we still want to install if a user installed kail somewhere else?

Reply:
I guess I should use `which`?

Reply:
agree with `which`